Sources/Rendering/Misc/RemoteView/index.js

```diff
@@ -219,6 +222,7 @@ export function extend(publicAPI, model, initialValues = {}) {
     'interactiveRatio',
     'stillQuality',
     'stillRatio',
+    'canvasElement',
```

Comment:
if no `canvasElement` is specified at construction time (i.e. as a parameter to `newInstance()`), `getCanvasElement()` will always return null. Shouldn't you update `model.canvasElement` when you create the canvas ?

Reply:
indeed that could be a good addon